app/src/main/res/layout/activity_contest_overview.xml

```diff
@@ -15,37 +15,28 @@
         android:text="@string/contest_overview_submit_button_text"
         />
 
-    <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginBottom="@dimen/activity_vertical_margin"
-                android:fillViewport="true"
-                android:paddingBottom="@dimen/activity_vertical_margin"
-                android:paddingLeft="@dimen/activity_horizontal_margin"
-                android:paddingRight="@dimen/activity_horizontal_margin"
-        >
-
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_above="@id/contest_overview_submit_button"
             android:orientation="vertical"
-            android:paddingTop="@dimen/activity_vertical_margin"
+            android:padding="@dimen/activity_vertical_margin"
```

Comment:
careful, is this intentional? You're tying the horizontal padding to the vertical margin.